extensions/amp-youtube/1.0/component.js

```diff
@@ -115,21 +114,55 @@ export function Youtube({
 
   src = addParamsToUrl(src, params);
 
+  /**
+   * These fields can be overridden by `infoDelivery` messages below.
+   * @const {{current: !JsonObject}}
+   */
+  const infoRef = useRef(
+    dict({
```

Comment:
This is allocated on every render, but only first one is used. An alternative, if you'd like it better, might be to do something like this:

```
const [info /* don't care about setter */] = useState(createDefaultInfo);

// everything else is the same.

...

// This will be called only once per component mount.
function createDefaultInfo() {
  return dict({...});
}
```